requirements.txt

```diff
@@ -1,4 +1,4 @@
-click==6.7
+click==7.0
```

Comment:
Please could you explain this change, and whether you have tested / looked for things that could break?

Reply:
I thought I was running into a click bug, where click.Path was not returning a Path. But, this, as you showed me was by design. I left it as 7.0 as all branch testing that has been done up to this point has been done with that version.

Reply:
Sounds good. Let's be careful because we don't have great CLI coverage and the API changed. Great to see that test coverage is being added so we can be more confident in this bump. I'm happy.